### PR TITLE
CB-10959 - integrate list-iteration based liftie call for liftie-only clusters

### PR DESCRIPTION
**Pull request by gregito**

### CODE REVIEW

File: environment/src/main/java/com/sequenceiq/environment/experience/liftie/LiftieIgnorableClusterStatuses.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    public boolean isEqualTo(String status) {
+        return name().equalsIgnoreCase(status);
+    }
+
+    public static boolean notContains(String status) {
+        return !contains(status);
+    }
+
+    public static boolean contains(String status) {
+        return !StringUtils.isEmpty(status) && Arrays.asList(LiftieIgnorableClusterStatuses.values())
```
Comment (bergerdenes) [inline environment/src/main/java/com/sequenceiq/environment/experience/liftie/LiftieIgnorableClusterStatuses.java:27]: Arrays.stream()

File: environment/src/main/java/com/sequenceiq/environment/experience/liftie/ExperiencesByLiftie.java [hunk 2]
```diff
@@ -39,18 +40,18 @@ public ExperiencesByLiftie(ExperienceIndependentLiftieClusterWorkloadProvider wo
...
     public void deleteConnectedExperiences(@NotNull EnvironmentExperienceDto environment) {
         throwIfTrue(environment == null, () -> new IllegalArgumentException(EnvironmentExperienceDto.class.getSimpleName() + " cannot be null!"));
         LOGGER.debug("Getting Liftie cluster list for environment '{}'", environment.getName());
-        List<ClusterView> clusterViews = getClusterViewsForAllProvidedWorkload(environment.getName(), environment.getAccountId());
+        List<ClusterView> clusterViews = getClusterViewsForWorkloads(environment.getName(), environment.getAccountId());
         LOGGER.debug("Starting Liftie clusters deletion for environment '{}'", environment.getName());
         clusterViews.stream()
-                .filter(cv -> LiftieIgnorableStatuses.notContains(cv.getClusterStatus().getStatus()))
+                .filter(cluster -> LiftieIgnorableClusterStatuses.notContains(cluster.getClusterStatus().getStatus()))
                 .forEach(cv -> liftieApi.deleteCluster(cv.getClusterId()));
```
Comment (bergerdenes) [inline environment/src/main/java/com/sequenceiq/environment/experience/liftie/ExperiencesByLiftie.java:55]: cv -> cluster to be consistent with the previous line

**Review by bergerdenes — CHANGES_REQUESTED** [review]

File: environment/src/main/java/com/sequenceiq/environment/experience/liftie/LiftieIgnorableClusterStatuses.java (hunk 1 above)
Comment (gregito) [reply environment/src/main/java/com/sequenceiq/environment/experience/liftie/LiftieIgnorableClusterStatuses.java]: done

File: environment/src/main/java/com/sequenceiq/environment/experience/liftie/ExperiencesByLiftie.java (hunk 2 above)
Comment (gregito) [reply environment/src/main/java/com/sequenceiq/environment/experience/liftie/ExperiencesByLiftie.java]: done

**Review by bergerdenes — APPROVED** [review]

**Pull request merged**